components/eam/src/physics/crm/physpkg.F90

```diff
@@ -629,6 +630,9 @@ subroutine phys_init( phys_state, phys_tend, pbuf2d, cam_out )
   ! Initialize ocean data
   if (has_mam_mom) call init_ocean_data()
 
+  ! Initialize MACv2-SP aerosols
+  call macv2_rad_props_init()
+
```

Comment:
should we add a check for do_macv2sp here?

Reply:
nevermind, I just noticed that you use the flag within that routine, which is ok.